docs/toolbox.md

```diff
@@ -0,0 +1,213 @@
+
+```js
+creator.toolbox
+  .getItemByName("dropdown")
+  .json
+  .choices = [
+    { text: "Option 1", value: 1 },
+    { text: "Option 2", value: 2 },
+    { text: "Option 3", value: 3 }
+  ];
+```
+
+[View Toolbox Customization example](https://surveyjs.io/Examples/Survey-Creator?id=toolboxcustomization)
+
+## Add a Custom Toolbox Item
+
+If you want to extend the predefined toolbox item collection, call the `addItem(itemConfiguration, [index])` method to add a custom item. This method accepts the following arguments:
+
+- `itemConfiguration`       
+A [toolbox item configuration object](https://surveyjs.io/Documentation/Survey-Creator?id=iquestiontoolboxitem). The [`name`](https://surveyjs.io/Documentation/Survey-Creator?id=iquestiontoolboxitem#name), [`iconName`](https://surveyjs.io/Documentation/Survey-Creator?id=iquestiontoolboxitem#iconName), and [`json`](https://surveyjs.io/Documentation/Survey-Creator?id=iquestiontoolboxitem#json) properties are required. When you configure an object for the `json` property, refer to the API help section of your question type for a list of available properties. For example, if you configure a Dropdown question, refer to the [Dropdown API](https://surveyjs.io/Documentation/Library?id=questiondropdownmodel) help section.
+
+- `index`       
+The target index of the item. Use `0` to add the item at the first position, `1` to add it at the second position, and so on. Do not specify `index` if you want to display the item at the end.
+
+The following code adds a custom item that allows users to select a country from a drop-down menu. Since the `index` argument is `0`, the toolbox displays the item at the beginning.
+
+```js
+creator.toolbox.addItem({
+  name: "countries",
+  iconName: "icon-dropdown",
+  title: "Countries",
+  json: {
+    type: "dropdown",
+    optionsCaption: "Select a country...",
+    choicesByUrl: {
+        url: "https://surveyjs.io/api/CountriesExample"
+    }
+  }
+}, 0);
+```
+
+[View Toolbox Customization example](https://surveyjs.io/Examples/Survey-Creator?id=toolboxcustomization)
+
+<!--
+
+WE HAVEN'T COME UP WITH A VERSION FOR REACT YET
+
+    ### Integrate 3rd-Party Components as Question Editors
+
+    Survey Creator supports integration with the following 3rd-party components out of the box:
+
+    %LIST%
+
+    To enable the integration with one of these components, reference or import the [surveyjs-widgets](https://github.com/surveyjs/custom-widgets) library next to the 3rd-party component sources:
```

Comment:
1. Please remove the article ("the") before "integration".

2. Since you say "reference or import the surveyjs-widgets library", I'm not sure what you mean by "next" in this context. Is it possible to import something _next_ to something else?